typings/balena-sdk/index.d.ts

```diff
@@ -873,6 +893,12 @@ export interface BalenaSDK {
 				): Promise<OrganizationMembership[]>;
 			};
 		};
+
+		hostapp: {
+			OsTypes: typeof OsTypes.OsTypes;
+			getAllOsVersions(deviceTypes: string[]): Promise<DeviceTypeOsVersions>;
+		};
```

Comment:
@thgreasi I wanted to keep it completely separate and mark it as beta just because we still don't have a good way to preload hostapps and releases in a non-balenaCloud environments.